src/libraries/System.Security.Cryptography.Xml/ref/System.Security.Cryptography.Xml.cs

```diff
@@ -11,43 +11,50 @@ public sealed partial class CipherData
         public CipherData() { }
         public CipherData(byte[] cipherValue) { }
         public CipherData(System.Security.Cryptography.Xml.CipherReference cipherReference) { }
-        public System.Security.Cryptography.Xml.CipherReference CipherReference { get { throw null; } set { } }
-        public byte[] CipherValue { get { throw null; } set { } }
+        public System.Security.Cryptography.Xml.CipherReference? CipherReference { get { throw null; } set { } }
+        public byte[]? CipherValue { get { throw null; } set { } }
         public System.Xml.XmlElement GetXml() { throw null; }
         public void LoadXml(System.Xml.XmlElement value) { }
     }
     public sealed partial class CipherReference : System.Security.Cryptography.Xml.EncryptedReference
     {
         public CipherReference() { }
-        public CipherReference(string uri) { }
-        public CipherReference(string uri, System.Security.Cryptography.Xml.TransformChain transformChain) { }
+        public CipherReference(string? uri) { }
+        public CipherReference(string? uri, System.Security.Cryptography.Xml.TransformChain transformChain) { }
         public override System.Xml.XmlElement GetXml() { throw null; }
         public override void LoadXml(System.Xml.XmlElement value) { }
     }
+    public partial class CryptoSignedXmlRecursionException : System.Xml.XmlException
```

Comment:
This type was declared `internal` in .NET Framework.  I don't know why it's `public` in our implementation here.  If the build fails with it declared internal in src we should just keep pruning it out of the ref.cs.

Reply:
Thanks. I pruned it!